### ISSUE TITLE
Daniel's site request

### ISSUE THREAD

**Issue by danimad** [rbind/support#42]
Hi,
I would like to try out rbind.

Currently I have a very bare bones site at danielb.netlify.com. I'd like to request the **danielberecz** subdomain.

Thanks!

**Comment by yihui** [comment]
danielberecz.rbind.io has been assigned to you.

**Comment by danimad** [comment]
Thanks!
I tried to set it up as a custom domain on Netlify there was an error message: "must be unique", and it wouldn't let me:
<img width="450" alt="netlify_error_message" src="https://user-images.githubusercontent.com/437923/27930473-fe842dea-6296-11e7-947e-fbea9b0f09cc.png">
Should I just wait, or do I something wrong? I read the documentation, and searched for an answer, but didn't find any.

**Comment by yihui** [comment]
Ah. You are not the first person who has run into this issue. I don't understand what this error message means. Could you delete the site on Netlify and create it again? Note you still have to use the Netlify site name `danielb`.

**Comment by danimad** [comment]
I tried that. I tried it now again just to be sure, but I've got the same error message.
Maybe it's a temporary error on their side, because while I experimented I got this error message:
<img width="400" alt="netlify_error_message_v2" src="https://user-images.githubusercontent.com/437923/27931000-f683692e-6298-11e7-91d2-1492cbc4ec6d.png">
The two message look like they are concatenated, I don't know how normal that is.
I will try again in the morning.

**Comment by yihui** [comment]
Could you do me a favor to contact their support? https://www.netlify.com/support/ Because I also want to know what is going on there. There were two other users in #40 who ran into the same issue. When I created the site for one of them, it worked fine on my side, but I don't want to do this for everyone every time.

**Comment by danimad** [comment]
I've sent an email detailing the problem. I will copy in the response when they answer.

**Comment by danimad** [comment]
The support people from Netlify wrote back, and said, that most likely their new [Branch Deploys](https://www.netlify.com/docs/continuous-deployment/#branch-deploys) feature is the culprit.
I quote: "Yihui has set up a site for 'rbind.io' on our service - which makes putting subdomain.rbind.io sites here a problem since if you called yours "staging" it would be mistaken for the staging branch build of rbind.io"
They said they will look into it more deeply. I wanted to share this now, but I'll keep you informed as things develop.

**Comment by yihui** [comment]
I see. That makes sense. I'm fine if the branch deploy feature is disabled for rbind.io.

**Comment by danimad** [comment]
Netlify fixed the issue for me. Yihui I added you to the email conversation, but I repeat what the people from Netlify said, for the others benefit. The issue is with the branch deploy feature as I described earlier. They can only set the rbind urls manually at the moment. They're working on a proper fix for the issue, but could't tell when it would be ready.
From my side, we can close this issue. If you agree I can close it.
Thank you for your help!
